data/ShopifyDemoData.xml

```diff
@@ -1,43 +1,36 @@
-    <moqui.service.message.SystemMessageRemote systemMessageRemoteId="DEMO_SHOPIFY_GET_FO_REMOTE" systemMessageTypeId="ShopifyGetFulfillmentOrderMessage" sendUrl="https://b6318f782c52bda768d23f5a404b51b6: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@moquidemo.myshopify.com/admin/api/2022-07/orders/" productStoreId="SHOPIFY_STORE" username="b6318f782c52bda768d23f5a404b51b6" password="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"/>
+    <!-- configuration for the demo store set up by Tailorsoft... possibly remove these values and let people enter their own -->
+    <moqui.service.message.SystemMessageRemote systemMessageRemoteId="ShopifyDemoRemote" systemMessageTypeId="ShopifySystemMessageType" 
+        productStoreId="POPC_DEFAULT" description="Remote for pushing and pulling data from the Demo Shopify store"
+        sendUrl="https://{store_name}.myshopify.com/admin/api/2023-01/${resource}.json" username="" password=""/>
 
-    <!-- PRODUCTS TO PROCESS SHOPIFY ORDERS-->
-    <products productName="Shopify_Demo 1" pseudoId="100000" productId="100000" productTypeEnumId="PtVirtual" ownerPartyId="ORG_ZIZI_RETAIL">
-        <features fromDate="1681493229737" amount="1" sequenceNum="1" productFeatureId="ColorBlack" applTypeEnumId="PfatDistinguishing"/>
-        <features fromDate="1681493256905" amount="1" sequenceNum="2" productFeatureId="ColorBlue" applTypeEnumId="PfatDistinguishing"/>
-    </products>
-    <products productName="Shopify Demo 1 black" pseudoId="Shopify_Demo_1_Bl" productId="100001" productTypeEnumId="PtVirtual" ownerPartyId="ORG_ZIZI_RETAIL">
-        <features fromDate="1681493229737" amount="1" sequenceNum="1" productFeatureId="ColorBlack" applTypeEnumId="PfatDistinguishing"/>
-        <features fromDate="1681493256905" amount="1" sequenceNum="2" productFeatureId="ColorBlue" applTypeEnumId="PfatDistinguishing"/>
-        <identifications productIdTypeEnumId="PidtVariantShopify" idValue="44890420576542"/>
-    </products>
-    <products productName="Shopify Demo 1 Blue" pseudoId="Shopify_Demo_1_Bu" productId="100002" productTypeEnumId="PtVirtual" ownerPartyId="ORG_ZIZI_RETAIL">
-        <features fromDate="1681493229737" amount="1" sequenceNum="1" productFeatureId="ColorBlack" applTypeEnumId="PfatDistinguishing"/>
-        <features fromDate="1681493256905" amount="1" sequenceNum="2" productFeatureId="ColorBlue" applTypeEnumId="PfatDistinguishing"/>
-        <identifications productIdTypeEnumId="PidtVariantShopify" idValue="44890420609310"/>
-    </products>
+    <!-- Service jobs to download Product and Collection data from Shopify -->
+    <moqui.service.job.ServiceJob jobName="poll_shopify_products_download" description="Poll download Shopify Products Demo"
+            serviceName="mantle.shopify.ShopifyProductServices.download#ShopifyProducts" cronExpression="0 0 * * * ?" paused="Y">

```

Comment:
Thinking about pulling data from shopify every hour. Is there a way that shopify can call update moqui on an update to their data like with a webhook or the like? If so that is probably the thing to aim for with a shopify-moqui "sync"

Reply:
There are webhooks avialable but would complicate greatly the integration.  See other people struggling with it here:
https://community.shopify.com/c/shopify-apis-and-sdks/webhook-for-product-s-collection-changing/td-p/449225

The purpose of the refactor was to cleanup the existing code to the best state possible, as simply as possible. I have stated in the README that the graph APi might be a better option than the REST API, but will wait for someone to actually ask for that before making that effort